bundles/binding/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/PLCLogoBinding.java

```diff
@@ -212,14 +208,14 @@ protected void execute() {
                         }
 
                         if (config.isSet()) {
```

Comment:
You detecting case when new value to be assigned by special value, and yes, in most cases it works as expected.

But author should remember that "magic", 
also if, for some reason, reported value is MAX_INT (in fact it is not invalid) it never will be assigned.